rllib/env/policy_client.py

```diff
@@ -172,6 +180,12 @@ def end_episode(self, episode_id, observation):
             "episode_id": episode_id,
         })
 
+    @PublicAPI
+    def update_policy_weights(self):
+        """Query the server for new policy weights, if local inference is enabled.
+        """
+        self._update_local_policy()
```

Comment:
Can add a force=True flag for immediate updates.

Reply:
Not sure what you mean?

Reply:
Oh just commenting that the interval check will skip the update potentially if the user is calling this manually. Hence, you might need a flag to force it to be synced.